web/packages/design/src/Icon/Icons/ShieldWarning.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* MIT License
```

Comment:
Is the double-license here correct? Is it mandated by the codegen tool?

Reply:
Yes this code was autogenerated. I believe it handles "our" license as well as the license to use the icon in this way? I'm not sure, but I'll leave it as the generator probably thought about it more than I did